server/services/workAndSkillsPageService.test.ts

```diff
@@ -182,17 +192,97 @@ describe('WorkAndSkillsService', () => {
     })
   })
 
-  describe('Scenario B', () => {
```

Comment:
I eventually worked out what "Scenario B" meant. It was tests (there was only 1 previously) that make assertions on the Curious Goals (previously known as learner goals) property of the Work And Skills service response object.

I've renamed and refactored the test to better reflect this, and added a couple more tests, including the new one that tests that `problemRetrievingData` is `true` when the Curious API fails

Reply:
Naming's hard, but it's easy not to call something so obscure!